cli/commands.go

```diff
@@ -14,7 +14,7 @@ var (
 			Name:      "list",
 			ShortName: "l",
 			Usage:     "List nodes in a cluster",
-			Flags:     []cli.Flag{flTimeout},
+			Flags:     []cli.Flag{flTimeout, flClusterStoreOpt},
```

Comment:
We should probably change all the occurrences mentioning `ClusterStoreOpt` or `ClusterOpt` to `DiscoveryOpt`.